app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -66,6 +67,12 @@ open class FenixApplication : Application() {
         // on the main process, as it uses Gecko to fetch experiments from the server.
         experimentLoader = loadExperiments()
 
+        // Enable the service-experiments component
+        Experiments.initialize(
+            applicationContext,
+            mozilla.components.service.experiments.Configuration(httpClient = lazy { components.core.client })
```

Comment:
If this will always be initialized on the main thread in service-experiments, then there's no need for the default lazy delegate thread safety. You could use `lazy(LazyThreadSafetyMode.NONE)`.

Reply:
Sounds reasonable to me.  Modification to remove the lazy delegate thread safety has been made.